testData/solutions/win32.json

```diff
@@ -1235,5 +1235,82 @@
                 "restoreSettings"
             ]
         }
+    },
+	
+    "org.chrome.cloud4chrome": {
+        "name": "Cloud4all Chrome extension",
+        "contexts": {
+            "OS": [{
+                    "id": "win32",
+                    "version": ">=5.0"
+            }]
+        },
+        "settingsHandlers": [
+           {
+                "type": "gpii.settingsHandlers.WebSocketsSettingsHandler",
```

Comment:
Also, you're missing the "options" block, take a look at https://github.com/GPII/universal/blob/master/testData/solutions/linux.json#L1451-L1453